client/google-search-app/src/app/app-google-footer/app-google-footer.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="footer-right">
```

Comment:
Can we use the file like this instead of mentioning google every folder:

       client/google-search-app/src/app/app-footer/app-footer.component.html